gh_actions_exporter/metrics.py

```diff
@@ -225,6 +203,7 @@ def handle_job_duration(self, webhook: WebHook, settings: Settings):
     def handle_job_cost(self, webhook: WebHook, settings: Settings):
         labels = self.job_labels(webhook, settings)
         # look for the flavor label
+
         flavor = labels.get(settings.flavor_label, None)
         cost_per_min = settings.job_costs.get(flavor, settings.default_cost)
         if webhook.workflow_job.conclusion and cost_per_min:
```

Comment:
This is where you should modify the logic, don't mess with the relabelling, let it do its thing, then you:
* checkout the labels inside webhook.workflow_job.labels
* Is there any that matches `settings.job_costs`
* Yes? Cool, let's calculate the cost with the value I have, and set it in the metrics
* No? Do nothing.